app/src/main/java/com/android/unio/model/preferences/AppPreferences.kt

```diff
@@ -0,0 +1,44 @@
+package com.android.unio.model.preferences
+
+object AppPreferences {
+  const val THEME = "theme"
+  const val NOTIFICATIONS = "notifications"
+  const val LOCATION_PERMISSION = "location_permission"
+  const val LANGUAGE = "language"
+
+  object Theme {
```

Comment:
Could it be useful to also add the options to change the contrast of the application if we also allow the user to change the theme here?

Reply:
Indeed, if needed, other themes can be easily added with this preference refactor.